### DISCUSSION TITLE
Expression of Interest in GSoC 2024 - Pybindings Project

### DISCUSSION THREAD

**Discussion by BryanEstrada003** [d-SEAMS/seams-core#45] [GSoC Discussions]
Hello d-SEAMS team,

I would like to join GSoC 2024 and, in particular, collaborate on the Pybindings project proposed by you. I have experience with PyVista, Matplotlib. How can I actively participate?

Thanks,
Michael Estrada

**Comment by HaoZeke** [comment]
Hi Michael, many apologies for the late response, it is great to hear your interest. I would suggest first building d-SEAMS and seeing if you can visualize the output files, the paper on JCIM has some examples of how it might look.
